Apps/W1/EDocumentConnectors/FORNAV/app/src/ForNAV/IncomingDoc.Table.al

```diff
@@ -0,0 +1,114 @@
+namespace Microsoft.EServices.EDocumentConnector.ForNAV;
+
+using Microsoft.EServices.EDocument;
+
+// Used by Azure function - do not modify
+table 6246260 "ForNAV Incoming Doc"
```

Comment:
Lets use a name that make is clear it is not part of incomming document feature, and something realted to API.